### PR TITLE
feature/static-page

### PR DESCRIPTION
**Pull request by przspa**
# Related issue

# Scope of work

[[deploy-preview]](https://deploy-preview-425--storefrontui-storybook.netlify.com/?path=/story/pages-static--default)
Realy easy page, base on SfContentPages component 

# Screenshots of visual changes
[desktop]
_d
[mobile][updated]

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by filrak** [comment]
image
Missing back button

**Comment by przspa** [comment]
@filrak, fixed but the back button should be a part of header I think.

File: packages/vue/src/examples/pages/static/Static.vue [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    margin: 0;
+    border: 0;
+    background-color: transparent;
+    outline: none;
+    cursor: pointer;
+  }
+  &__title {
+    font-family: $body-font-family-secondary;
+    font-size: $font-size-big-mobile;
+    font-weight: 500;
```
Comment (leomp12) [inline packages/vue/src/examples/pages/static/Static.vue]:
```suggestion
    font-weight: $h2-font-weight-desktop;
```

Btw, we really need [all that variables](https://github.com/DivanteLtd/storefront-ui/blob/develop/packages/shared/styles/variables/_typography.scss#L43) for font weight @filrak @przspa ?
Is hard to decide sometimes, why not just `$font-weight-normal` and `$font-weight-title`?

I think almost all of these variables are used only on [`_headings.scss`](https://github.com/DivanteLtd/storefront-ui/blob/develop/packages/shared/styles/global/_headings.scss), but the way it is used (styles directly for Hn tags) could be easily defined by the user with simple CSS globally.

Comment (przspa) [reply packages/vue/src/examples/pages/static/Static.vue]: we used font-weight 300 (lite), 400 (normal) and 500 (medium)

**Comment by przspa** [comment]
tanks @leomp12 I improvement your request changes

File: packages/vue/src/examples/pages/static/Static.vue (hunk 1 above)
Comment (leomp12) [reply packages/vue/src/examples/pages/static/Static.vue]: @przspa right, so maybe `$font-weight-muted`, `$font-weight-base` and `$font-weight-title`?
I just thing we don't need one per heading tag and device.
Even to facilitate user customization

**Comment by przspa** [comment]
thanks for CR @leomp12 Can you add issue for this scss variable refactor?

**Pull request merged**